abtesting/Shared/UITests.swift

```diff
@@ -68,7 +68,13 @@ class UITests: XCTestCase {
       let texts = ["Getting Started with Firebase", "An Introduction to Firebase"]
     #endif
     for text in texts {
-      XCTAssertTrue(app.staticTexts[text].isHittable, "Text '\(text)' is missing from view.")
+      let element = app.staticTexts[text]
+      // TODO: This if should be removed.
```

Comment:
Flagging this in case you meant to remove before merging.

Reply:
This is from https://github.com/firebase/quickstart-ios/pull/1456. Tests fail without it on GHA but not locally. I'll try if it's fixed on Xcode 14 though.

Reply:
Still present on Xcode 14 :(